src/libsyntax/diagnostic_list.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2014 The Rust Project Developers. See the COPYRIGHT
+// Copyright 2016 The Rust Project Developers. See the COPYRIGHT
```

Comment:
Should the copyright year be updated here? Should it be saying 2016 and not 2014-2016? Is the convention for this recorded anywhere?